packages/core-ui/lib/core/index.tsx

```diff
@@ -12,7 +12,10 @@ import { EntryProps, PageRowProps, SideProps } from "./types";
  * The outer-most wrapper that hosts core tag styling.
  */
 export const Entry = ({ children, style, mode, network }: EntryProps) => (
-  <div className={`core-entry theme-${mode} theme-${network}`} style={style}>
+  <div
+    className={`core-entry theme-${mode} theme-${network} page-padding`}
```

Comment:
```suggestion
    className={`core-entry theme-${mode} theme-${network}`}
```

This component does not use page-padding, it just defines the class.